compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/tree/STFailStatementNode.java

```diff
@@ -30,62 +30,66 @@
  *
  * @since 2.0.0
```

Comment:
version should be Swan Lake

Reply:
Actually, we have decided to move ahead with 2.0.0. Let's use "2.0.0" instead of Swan Lake.